pymeasure/instruments/activetechnologies/AWG401x.py

```
@@ -460,6 +460,43 @@ def __init__(self, adapter, **kwargs):
+        ("SOURce2:VOLTage:LEVel:IMMediate:AMPLitude? MINimum", "VPP1"),
+        ("SOURce2:VOLTage:LEVel:IMMediate:AMPLitude? MAXimum", "VPP2"),
+        ("SOURce2:VOLTage:LEVel:IMMediate:OFFSet? MINimum", "1"),
+        ("SOURce2:VOLTage:LEVel:IMMediate:OFFSet? MAXimum", "2"),
+        ("SOURce2:VOLTage:BASELINE:OFFSET? MINimum", "1"),
+        ("SOURce2:VOLTage:BASELINE:OFFSET? MAXimum", "2"),
+        ("SOURce2:FREQuency? MINimum", "1"),
+        ("SOURce2:FREQuency? MAXimum", "2"),
+        ("SOURce2:PHASe:ADJust? MINimum", "1"),
+        ("SOURce2:PHASe:ADJust? MAXimum", "2"),
+        ("*IDN?", "x,AWG4012"),
+    ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)